ONSdigital/dp-frontend-search-controller
# Add AB test handlers for nlp weighting

### What

Added AB testing functionality for NLP weighting. 

### How to review

Check tests pass, code looks good. 

Optionally:
- port forward to api router
- set ab_test cookie to appropriate value
- see the requests coming in to search-api via kibana with nlp_weighting=true vs false
- check the gtm data layer script to see different values being assigned to "abTest"

### Who can review

Frontend Go dev.
